src/IQFeed.CSharpApiClient/Streaming/Derivative/DerivativeClient.cs

```diff
@@ -63,11 +64,21 @@ public void Connect()
             _socketClient.Connect();
         }
 
+        public async Task ConnectAsync()
+        {
+            await _socketClient.ConnectAsync();
```

Comment:
When possible, use plain return of the Task instead of await and async.
Using async/await adds overhead behind the scenes so when its better to avoid when possible.

Reply:
Updated